walkthroughs/howto-k8s-ingress-gateway-enhancements/README.md

```diff
@@ -39,6 +39,8 @@ The basic setup will create an ingress-gw and a gatewayroute that can route traf
 ```
 The virtual service will be targetting 2 virtual nodes blue and red via virtual router and depending on the path it will route traffic to the corresponding node.
 
+So until now, we had to make use of a VirtualRouter to further distinguish routes based on different paths. Now we have added this functionality to the VirtualGateway itself and we don't need to make use of an intermediate VirtualRouter. 
```

Comment:
So, VR is still useful for traffic within mesh and the above sounds as if VR isn't really necessary anymore if VG is in place. We can instead probably say something around the lines of "VirtualGateway now supports path, header and query based routing. Below series of steps will provide a sample overview of how to make use of these features".  We can also let VG to rewrite paths and underlying VR can be matching on those as well...